index.html

```diff
@@ -274,50 +294,67 @@ <h3>Namespace Specific Identifier (NSI)</h3>
+              <p>Peer DIDs must be compared according to whatever case-sensitivity is implied by the
+              hash encoding they use. Hex is not case-sensitive, but if a future version of the spec
+              uses base58 or base64 encoding, values will be case sensitive. Therefore, it is strongly
+              recommended that routines that handle peer DIDs not normalize case unless they take into
+              account the specific hash encoding.</p>
+              <p>As with git commit hashes, it is likely that a small subset of the bits in the <em>numeric basis</em>
+                  of a peer DID will be sufficiently unique for some contexts, and will be more convenient for human
+                  interaction. Therefore, a lossy <em>short form</em> of a peer DID is recognized if the hash encoding
+                  is hex. This form must consist of at least 4 bytes (8 hex characters) after the multibase and
+                  multihash prefixes. It would be suitable for storing in a log file, echoing in a UI, and so forth.
+                  It may NOT be used where precision is required (e.g., in a DID Doc, in DIDComm routing, in credentials).
               </p>
         </section>
         <section>
             <h3>Namestring Generation Method</h3>
-            <p>The unique numeric value underlying a Peer DID MUST be generated by using a cryptographically robust
-                algorithm with secure randomness to create an Ed25519 keypair (or, when additional formats are supported,
-                an analogous keypair on a different curve), and then selecting as the NSI a subset of the public
-                verification key that is at least 128 bits, and that is dependent on <code>keyfmtchar</code>. For
-                <code>keyfmtchar</code> = "1", the NSI = the most significant 16 bytes of the public verification key.</p>
-            <p>In this way, the DID can be known to begin its existence already associated with keys, and the owner
-                of the new DID is guaranteed to be the only entity in the world who possesses the private key.
-                This guarantees the initial integrity of the DID's chain of custody.
-            </p>
+            <p>The unique <em>numeric basis</em> underlying a Peer DID MUST be generated as follows:</p>
+            <ul>
+                <li>Create a <em>genesis</em> version of the DID Doc for the DID. The genesis version must include at
+                least one verification key.</li>
+                <li>Remove the main <code>id</code> field (the one that would normally contain the DID value) from the DID Doc.
+                This creates a variant of the peer DID doc known as the <em>stored variant</em>, as opposed to the
+                <em>resolved variant</em> that would have an actual DID value in the <code>id</code> field.</li>
+                <li>Calculate the SHA256 hash of the bytes of the stored variant of the DID Doc, and select the first 16
```

Comment:
So are we truncating the SHA256 hash as the DID? In that case the multihash designation will not technically be correct since the hash length and algorithm does not really match. Further down the example DID is 32 bytes.

Reply:
Whoops. I meant to remove all mention of trucation. I thought I had caught them all, but I guess this one slipped past me. No, we shouldn't be truncating anymore. I'll fix this.

Reply:
Ok sounds good.